Add song 'Die Perfekte Diktatur' by 'Farin Urlaub Racing Team'

Quizznor opened the pull request.

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+\beginsong{Die perfekte Diktatur}[by={Die Ärzte}]
```
I am not that deep in the Ärzte game but shouldn't this be `by={Farin Urlaub Racing Team}`.
For me it's not a big deal, but for others it might be, so IMO this should be changed.
Reference: https://de.wikipedia.org/wiki/Farin_Urlaub

Note: If we decide to change this, we should also update the parent folder.

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+\beginsong{Die perfekte Diktatur}[by={Die Ärzte}]
+
+    \textnote{Your guitar should be tuned to an E$\,\flat$}
```
Side note: I think one wouldn't drop the tuning of the guitar while playing songs from this songbook with friends or at camp fire. But we should still leave this comment as recommendation.

@dlik is the text note clear for you or should there be more info?

dlik replied:
How about we just write down the full tuning? Like this:

` \textnote{Tuning: E$\,\flat$ A$\,\flat$ D$\,\flat$ G$\,\flat$ B$\,\flat$ E$\,\flat$}`

dlik replied:
Or alternatively: 
`\textnote{Tune half step down}`

hd1ex replied:
> How about we just write down the full tuning? Like this:
> 
> ` \textnote{Tuning: E$\,\flat$ A$\,\flat$ D$\,\flat$ G$\,\flat$ B$\,\flat$ E$\,\flat$}`

Yeah I'm in favor of this

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+\beginsong{Die perfekte Diktatur}[by={Die Ärzte}]
+
+    \textnote{Your guitar should be tuned to an E$\,\flat$}
+    Intro: Am Em Fmaj7 Fadd9 (2x)
```
Maybe we should use `\rep{2}` for this.

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+        \[Am] Ich fahr nicht mit der Strassen\[Em]bahn,
+        Ich lauf den ganzen Weg zu \[Fmaj7]dir. \[Fadd9]
+        \[C] Das Telefon wird abge\[G]hört
+        und in der Bahn sitzen Spi\[Dm]one. \[Dsus]
```
Which `Dsus` is meant here?
Please specify with `Dsus2` or `Dsus4`.

Quizznor replied:
I'm not sure. I'm thinking the website I pulled the riff from isn't entirely trustworthy.
Rewatching [this video](https://www.youtube.com/watch?v=LB57MEDteck) I only see 
Farin Urlaub playing a `Dm`. Will delete all `Dsus` chords accordingly.

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+    \[Am] Wir planen schon seit einem Jahr \[Em]
+    zusammen von hier abzu\[Fmaj7]hau'n. \[Fadd9]
+    \[C]Du hast uns Dynamit bes\[G]orgtt,
+    jetzt brauchen wir noch ein paar \[F]Waffen. \[Dm] \[Dsus]
```
Dito regarding `Dsus`

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+      \[Am] Dein Name stand nicht an der \[Em]Klingel
+      in der Tür stand \[Fmaj7]irgendwer. \[Fadd9]
+      \[C] Er gab mir einen Brief von \[G]dir
+      in dem du schriebst es tut dir \[F]Leid. \[Dm] \[Dsus]
```
Dito regarding `Dsus`

hd1ex reviewed (changes requested):
Over all the song LGTM.
I added some minor suggestions which have to be addressed.

Furthermore I would really like to have this song on two pages.
Can we maybe reuse a chorus like it was done [here](https://github.com/hd1ex/songs/blob/master/songs/imagine-dragons/radioactive.tex)?

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+      Überall sind \[Dm]Kameras angebracht,\[Am]
+      alles wird \[C]rund um die Uhr überwacht. \[G]
+      Die Grenzen sind ge\[Dm]schlossen seit Jahren schon,\[E]
+      dir bleibt nur noch die \[Am]innere \[Gm]Emigra\[Am]tion.
```
The `Gm` should be an `G`

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+      Überall sind \[Dm]Kameras angebracht,\[Am]
+      alles wird \[C]rund um die Uhr überwacht. \[G]
+      Die Grenzen sind ge\[Dm]schlossen seit Jahren schon,\[E]
+      dir bleibt nur noch die \[Am]innere \[Gm]Emigra\[Am]tion.
```
Dito regarding `Gm`

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+      Überall sind \[Dm]Kameras angebracht,\[Am]
+      alles wird \[C]rund um die Uhr überwacht. \[G]
+      Die Grenzen sind ge\[Dm]schlossen seit Jahren schon,\[E]
+      dir bleibt nur noch die \[Am]innere \[Gm]Emigra\[Am]tion.
```
Dito regarding `Gm`

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+    \beginchorus
+      \[Dm]Sie kommen meistens nachts um die Leute zu \[Am]holen,
+      und hinterher steht \[C]wieder eine Wohnung leer. \[G]
+      Die Zeitungen sind \[Dm]voll mit Durhaltepa\[Am]rolen,
```
Typo:
Durhalteparolen -> Durchhalteparolen

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+    \beginchorus
+      \[Dm]Sie kommen meistens nachts um die Leute zu \[Am]holen,
+      und hinterher steht \[C]wieder eine Wohnung leer. \[G]
+      Die Zeitungen sind \[Dm]voll mit Durhaltepa\[Am]rolen,
```
Typo:
Durhalteparolen -> Durchhalteparolen

hd1ex commented on songs/die-aerzte/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,96 @@
+    \beginchorus
+      \[Dm]Und wir kommen meistens nachts um die Leute zu \[Am]holen,
+      und hinterher steht \[C]wieder eine Wohnung leer. \[G]
+      Die Zeitungen sind \[Dm]voll mit Durhaltepa\[Am]rolen,
```
Typo:
Durhalteparolen -> Durchhalteparolen

hd1ex reviewed (changes requested):
Some things I noticed during playing/asking a friend.

Quizznor commented:
> Over all the song LGTM.
> I added some minor suggestions which have to be addressed.
> 
> Furthermore I would really like to have this song on two pages.
> Can we maybe reuse a chorus like it was done [here](https://github.com/hd1ex/songs/blob/master/songs/imagine-dragons/radioactive.tex)?

I have updated the song structure accordingly. I can shorten the length even further if you desire. Otherwise I think it would be beneficial to have the full chord progression of the chorus on the second page as well. There is enough space anyway.

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,81 @@
+    ich weiß ich kann dir auch ver\[Fmaj7]trau'n \[Fadd9]
+    \[Am] Wir planen schon seit einem Jahr \[Em]
+    zusammen von hier abzu\[Fmaj7]hau'n. \[Fadd9]
+    \[C]Du hast uns Dynamit bes\[G]orgtt,
```
This is probably a typo:
besorgtt -> besorgt

hd1ex reviewed (changes requested):
> Otherwise I think it would be beneficial to have the full chord progression of the chorus on the second page as well. There is enough space anyway.

Yeah that's true.
Also I would suggest putting the to be played choruses in their own chorus envs like so:
```
    \begin{chorus}
        \textbf{Chorus}
    \end{chorus}
```
So someone playing it can differentiate between _"this is the chorus"_ and _"now play the chorus"_.

Lastly: Please **squash** these commits into one (GitHub already tracks the history of this PR).

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    ich werd wohl nicht viel \[Fmaj7]schlafen. \[Fadd9]
+    \endverse
+
+    \beginchorus
```
Since you reference a **chorus** on the same page: Is this the chorus that has to be replayed (there are 2 defined)?
If so I'd add a marking like this:
```suggestion
    \beginchorus
       \textbf{Chorus}
```
People then can reference this immediately.

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endchorus
+
+    \beginverse
+    \[Am] Ich kenn dich seit wir Kinder \[Em]sind,
```
Your indentation is inconsistent here: Please use 4 spaces like everywhere else

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endverse
+
+    \beginchorus
+      \[Dm]Sie kommen meistens nachts um die Leute zu \[Am]holen,
```
Same regarding indentation

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endchorus
+
+    \beginchorus
+      Die Grenzen sind ge\[Dm]schlossen seit Jahren schon,\[E]
```
Same regarding indentation

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endverse
+
+    \beginchorus
+      \[Dm]Sie kommen meistens nachts um die Leute zu \[Am]holen,
```
Same regarding indentation

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endchorus
+
+    \beginchorus
+      Trotzdem ist es \[Dm]heute nacht wieder pass\[Em]iert.
```
Same regarding indentation

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endchorus
+
+    \beginverse
+      \[Am] Ich ging auch heut wieder zu \[Em]dir,
```
Same regarding indentation

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endverse
+
+    \beginchorus
+      \textbf{Chorus}
```
Same regarding indentation

hd1ex commented on songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex:
```diff
@@ -0,0 +1,92 @@
+    \endchorus
+
+    \beginchorus
+      Die Grenzen sind gesch\[Dm]lossen seit Jahren schon. \[Em]
```
Same regarding indentation

hd1ex reviewed (changes requested):
This PR is in a good shape now: Thanks for you patience!
Here are some comments regarding style and playability.

hd1ex commented:
`make songbook` currently puts this song on the second column of the previous song.
This is not intended and should be addressed somewhere.
Don't know how to fix this tho.

Quizznor committed:
Add Song 'Die Perfekte Diktatur' by 'Farin Urlaub Racing Team'

hd1ex commented:
LGTM

dlik committed:
Change some chords in 'Die perfekte Diktatur' by 'Farin Urlaub Racing Team'

hd1ex commented:
LGTM
